WTWSMS/history/wars/gothic_war.txt

```diff
@@ -1,19 +1,38 @@
-name = "The Gothic Wars"
+name ="Gothic War"
 
 casus_belli = {
-	casus_belli = unification 
-	actor = 70512 
-	recipient = 190215 
+	casus_belli = unification
+	actor = 70512
+	recipient = 190215
 	landed_title = e_italy
 	date = 535.1.1
 }
 
-476.1.1 = {
-	add_attacker = 70512 
-	add_defender = 190215 
+535.1.1 = {
+	add_attacker = 70512
+	add_defender = 190215
 }
-
-476.8.15= {
-	rem_attacker = 70512 
-	rem_defender = 190215 
+536.1.1={
+	add_defender = 190219 # Witiges
+	rem_defender = 190215
+}
+540.1.1={
+	add_defender = 190222 # Ildibad
+	rem_defender = 190219
+}
+541.1.1={
+	add_defender = 190116 # Eraric
+	rem_defender = 190222
+}
+543.1.1={
+	add_defender = 190118 # Totila
+	rem_defender = 190116
+}
+552.1.1={
+	add_defender = 155147 # Thila
+	rem_defender = 190118
+}
+553.1.1 = {
+	rem_attacker = 70512
+	rem_defender = 155147
```

Comment:
I think the intent of this file might be something else. I'd rather not change it but rather add a new one.

Reply:
OK. So is it kind of a workaround war?

Reply:
OK. The old version is back (in gothic_war_old.txt).

Reply:
Okay.